data/os/FreeBSD.yaml

```diff
@@ -2,7 +2,7 @@
 mcollective::plugin_group: wheel
 mcollective::configdir: "/usr/local/etc/choria"
 mcollective::bindir: "/usr/local/bin"
-mcollective::libdir: "/usr/local/share"
+mcollective::libdir: "/usr/local/share/mcollective/plugins"
```

Comment:
I am not sure about what to put here…  Maybe `/usr/local/share/choria/plugins` makes more sense… managing files into `/usr/local/share/choria/plugins/mcollective` which could see the `mcollective` component disappear at some point in the future?

What do you think?

Reply:
I think the mcollective bit will stay for a long time - ruby does “require ‘mcollective/agent/foo’” would be a sizeable change one that I do want to make but for now maybe not

Reply:
share/choria as linode seems good in time when I change it the purge will restructure it